frontend/e2e/cypress/support/page_objects/NavigationConstructor.js

```diff
@@ -59,11 +59,11 @@ export class NavigationConstructor {
     return this;
   }
   methodologyPage() {
-    navigateDropdownMenuItem("a", "Methodology", "/methodology");
+    handleMenuExternalLink("Methodology", "https://github.com/climatepolicyradar/methodology");
```

Comment:
This link also shows some "Code Book" :shrug: 
Should this actually be: https://github.com/climatepolicyradar/methodology/blob/main/METHODOLOGY.md

Reply:
resolving to just get this merged